core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -69,9 +69,16 @@
 /* Normal memory, Outer Write-Back Write-Allocate Cacheable */
 #define TEE_MMU_TTB_RNG_WBWA    (1 << 3)
 
+/* Normal memory, Outer Write-Back no Write-Allocate Cacheable */
+#define TEE_MMU_TTB_RNG_WB      (3 << 3)
+
+#ifndef CFG_NO_SMP
```

Comment:
Please fix ARMv8 and ARMv7 with LPAE also, or at least introduce a compile error if `CFG_NO_SMP` is set.